### PR TITLE
Feat:setup page + routing + compilers etc

### PR DESCRIPTION
**Pull request by zoepage**
setup:
- templates
- routing
- basic compiling (scss, md, njk, liquid)
- fonts
- images
- RSS feed
- docs folder for deployment for GH-pages
and more...

````bash
npm install
npm start
````

### CODE REVIEW

File: docs/404.html
Comment (GiordanaFurquim) [inline docs/404.html:1]: should we format this file? 🙊

File: src/assets/css/main.scss [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+  @font-face {
+    font-family: "AVF";
+    font-style: light;
+    font-weight: 300;
+    font-display: auto;
+    src: url("../fonts/AVF.ttf");
+  }
+
+//   background: linear-gradient(0deg, #F1DE00, #F1DE00),
+// linear-gradient(0deg, #000000, #000000),
```
Comment (GiordanaFurquim) [inline src/assets/css/main.scss:18]: Hey, can you please remove these commented out lines?

File: src/assets/css/prism-base16-monokai.dark.scss [hunk 2]
```diff
@@ -0,0 +1,89 @@
+code[class*="language-"], pre[class*="language-"] {
+	font-size: 14px; 
+	line-height: 1.375;
```
Comment (GiordanaFurquim) [inline src/assets/css/prism-base16-monokai.dark.scss:3]: indentation: one space to much in all file :D

File: src/assets/css/reset.scss [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+b, u, i, center,
+dl, dt, dd, ol, ul, li,
+fieldset, form, label, legend,
+table, caption, tbody, tfoot, thead, tr, th, td,
+article, aside, canvas, details, embed, 
+figure, figcaption, footer, header, hgroup, 
+menu, nav, output, ruby, section, summary,
+time, mark, audio, video {
+	margin: 0;
+	padding: 0;
```
Comment (GiordanaFurquim) [inline src/assets/css/reset.scss:20]: check indentation :D

File: src/assets/css/main.scss (hunk 1 above)
Comment (zoepage) [reply src/assets/css/main.scss:18]: sure! :) i left them for raffa so he would know how I did the circle that we talked about. but as he as forked this anyway, i am happy to delete :)

File: src/assets/css/prism-base16-monokai.dark.scss (hunk 2 above)
Comment (zoepage) [reply src/assets/css/prism-base16-monokai.dark.scss:3]: we need to setup a linter & formatter. would you mind creating the ticket for it?

Comment (GiordanaFurquim) [reply src/assets/css/prism-base16-monokai.dark.scss:3]: There we go: https://github.com/deven-org/deven-website/issues/29 <3

**Review by GiordanaFurquim — APPROVED** [review]

**Pull request merged**